react-native/setup/setup_firebase_multiple_envs.mo.md

```diff
@@ -0,0 +1,66 @@
+# [MO] Setup Firebase on iOS/Android with multiple environments *(~1h)*
+
+## Owner: Louis Lagrange
+
+## Prerequisites *(~30min)*
+
+- [ ] An Android or iOS app with the Firebase SDK installed
+   - For React Native apps, you can use [`react-native-firebase`](https://github.com/invertase/react-native-firebase)
+
+## Steps *(~30min)*
+
+It's pretty straightforward to install Firebase when you have only one environment.
+But what if you have multiple environments, such as `dev`, `staging` and `production`?
+Here's the steps by steps.
```

Comment:
C'est correct de dire "Here's the steps by steps"? Peut être plutôt "Here's how" ou "Here are the steps"? :)